packages/react-slider/src/components/RangedSlider/useRangedSliderStyles.ts

```diff
@@ -0,0 +1,168 @@
+import { mergeClasses, makeStyles } from '@fluentui/react-make-styles';
+import {
+  thumbClassName,
+  trackClassName,
+  useRootStyles,
+  useSliderWrapper,
+  useRailStyles,
+  useTrackWrapperStyles,
+  useTrackStyles,
+  useMarksWrapperStyles,
+  useThumbWrapperStyles,
+  useThumbStyles,
+  useActiveRailStyles,
+} from '../Slider/useSliderStyles';
+import { createFocusIndicatorStyleRule } from '@fluentui/react-tabster';
+import type { RangedSliderState } from './RangedSlider.types';
+
+export const lowerThumbClassName = mergeClasses(thumbClassName, 'lower');
+export const upperThumbClassName = mergeClasses(thumbClassName, 'upper');
```

Comment:
These should be more specific than just `lower` and `upper` to allow potential collisions

```suggestion
export const lowerThumbClassName = mergeClasses(thumbClassName, `${thumbClassName + '-lower'}`);
export const upperThumbClassName = mergeClasses(thumbClassName, `${thumbClassName + '-upper'}`);
```